runtime/js/99_main.js

```diff
@@ -1,3 +1,4 @@
+"use strict";
```

Comment:
Not sure if this file should be in strict mode because according to MDN, undeletable variables like `__proto__` can't be deleted in strict mode.

Reply:
Tests did pass with this though...